ss/multiconn.go

```diff
@@ -0,0 +1,25 @@
+package ss
+
+import (
+	"strings"
+	"sync"
+)
+
+var routes struct {
+	sync.RWMutex
+	Target2Client map[string]string // map target ip to local tunnel port
+	DefaultClient string            // the default client for the targets are not in Target2Client map
+}
+
+func getClient(target string) string {
+	tgtIp := strings.Split(target, ":")

```

Comment:
Is it possible that multiple targets share the same IP address?

Reply:
ss use IP and port to set up connections. Currently it seems difficult to distinguish same IP addresses at ss level although we can distinguish them when `getRemoteInfo` .